### ISSUE TITLE
[Bug] Slotting Generator HATES Overflow Slots, CLICK NOW TO FIND OUT WHY

### ISSUE THREAD

**Issue by dgibso29** [Bubbus/F3_CA_BUB#228]
**Describe the bug**
Slotting generator is bamboozled when crafty Mission Makers SLAM role descriptions with "(Overflow)" and fails to create an appropriate short code.

**To Reproduce**
1. Make slot with `(Overflow)` in the role description.
2. Generate slotting post
3. Behold the long short code

**Expected behavior**
Parentheses don't break Short Code generation, or at the very least the common "(Overflow)" pattern is handled.

**Screenshots**
Slotting Output
![image](https://github.com/user-attachments/assets/d275c706-32b9-4c7b-8ff9-56955b8a71f3)

**Environmental Info(please complete the following information):**
 - V+
 - WS; RF;
 - 1.7.0

**Additional context**
Probably just an issue with parsing parentheses or other special characters. Not exactly a huge deal to fix in the output but could confuse people that aren't us.

**Comment by Bubbus** [comment]
Is this a documentation issue?  Does this help?
https://github.com/CombinedArmsGaming/CAFE3/blob/develop/configuration/slottingGenerator.hpp#L37

**Comment by dgibso29** [comment]
Whoopsie, nope; I simply forgot about this note.

FWIW: I saw this as a problem stemming from Parentheses. It might be worth looking for an ignoring "(Overflow)" in general.

otherwise skill issue accepted

**Comment by dgibso29** [comment]
Yeah in retrospect I think it makes sense to trim out "(Overflow)" and "Overflow" rather than asking MMs to update this file or manually update their slotting (more annoying than it sounds to have to go fix it 9 times)
